composer.json

```diff
@@ -49,9 +49,13 @@
     "require-dev": {
         "silverstripe/frameworktest": "4.x-dev",
         "silverstripe/graphql-devtools": "1.x-dev",
-        "silverstripe/recipe-testing": "^1",
+        "silverstripe/recipe-testing": "^2",
         "mikey179/vfsstream": "^1.6",
-        "sminnee/phpunit-mock-objects": "^3.4.7"
+        "dms/phpunit-arraysubset-asserts": "^0.3.0"
+    },
+    "replace": {
+        "sminnee/phpunit": "*",
+        "sminnee/phpunit-mock-objects": "*"
     },
```

Comment:
Why do we need this?

Reply:
Some old modules still have sminnee/phpunit in there, this ensures that 'phpunit/phpunit' "wins". Merging this as-is will make it easier to remove it later.

We'll have to remove the `replace` later once we've fixed those other modules.